### ISSUE TITLE
Pls allow to combine VASE mode and normal print mode / different objects

### ISSUE THREAD

**Issue by DM4DS** [bambulab/BambuStudio#1002]
Problem: I´m coming from a commercial slicer and there I am easily able to combine VASE mode and "normal" print mode in one object by spliting the job in 2 sections.
This is not possible in Bambulab slicer. Tryed to "cheat" with multicolor, spliting objects , loading 1 part for the vase mode job + 1 for the normal printing but the slicer drops the second, which should be at height x, to the build plate and still I´m not able to enable vase for one and normal for the other.

Solution: Alow, when splitting the part in 2 objects (or more) to enable vase mode in one and doing a normal print job in the other(s).

My alternative:
I´m exporting gcode right now and edit 2 files into 1, copy it on the sdcard and print the job, but that´s annoying.

Content:
You can watch my "help" video on video. There you also see the printed part from an older , not optimized profile, combined vase/normal printing, yes there are parts were this is extremly helpfull and safes a ton of time and material:
https://youtu.be/qdB6qTi6eeA

**Comment by barrettj** [comment]
I don't know if you'll be able to continue on with the vase mode part, but you can raise an object if it's part of an assembly

If you select both objects and "assemble" them, you can then choose just one part form the object list and raise it.

![Screenshot 2023-01-05 144216](https://user-images.githubusercontent.com/361635/210876122-967deab5-ceba-460c-a202-4248915b9b57.png)

![Screenshot 2023-01-05 144148](https://user-images.githubusercontent.com/361635/210876155-f4ad640b-1560-445d-8b14-56a7868c8e3e.png)

**Comment by mkosmo** [comment]
Vase mode does a continuous spiral instead of the discrete layers of non-vase mode, so I'm not sure how the two modes would be compatible simultaneously.

**Comment by DM4DS** [comment]
Thanks for reply.Look into simplify3d , there it is easy to do and I print parts since years in this combination.Can send you videos....GreetingsSandor Dubronner -- Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.

 
  
  
 
 Am 07.01.23, 04:08 schrieb Matthew Kosmoski ***@***.***>:

   
  Vase mode does a continuous spiral instead of the discrete layers of non-vase mode, so I'm not sure how the two modes would be compatible simultaneously.

**Comment by Malitybo** [comment]
I agree this would be a great addition. I use it to create lampshades. I use the bottom layers for a sturdy base of the shade from which it hangs and then the follow-up layers in vase mode which are for aesthetics. I still use this on my ultimaker in combination with simplify3d, but would love to use my bambulab printer for these jobs. Any clues if this feature will be added somewhere soon?

**Comment by michaelwschultz** [comment]
+1 Would love to see this as well. having the ability to use vase mode starting at a specific layer can really help with print times as well as quality of certain prints. Hope this feature makes it into the pipeline soon.

**Comment by ndonathan** [comment]
I'd love flexibility in being able to do this, but just being able to transition from layer printing to vase mode at a specific layer would work in all my use cases. I wouldn't need the ability to switch modes more than once and vase mode would always be on the upper layers.

**Comment by m9tdev** [comment]
I've found a solution/workaround for this problem (as long as the first part fits under the print arm when the toolhead is on the plate): I wanted to print a base with PLA Basic and a wall on top of it using PETG Translucent using Spiral Vase mode.

Step 1:
Make sure the object is in a place that doesn't interfere with toolhead cleaning and homing (e.g: bottom-right, as long as the first object fits below the print arm, otherwise this probably doesn't work).

<img width="3188" height="2110" alt="Image" src="https://github.com/user-attachments/assets/1dae094a-5330-4397-8105-cd166128542b" />

Step 2:

Select `Objects`, select the part you want to print on top later, and move it below the plate.

<img width="1594" height="1055" alt="Image" src="https://github.com/user-attachments/assets/0569cd64-20a9-4ecd-a7b2-24cf4106f101" />

Now you can print that part with the settings you like.

Step 3:

Revert moving the part below the plate, and instead now move the first/base part below the plate.

<img width="3188" height="2110" alt="Image" src="https://github.com/user-attachments/assets/691390c5-1c3e-47f6-81d6-6638a321fcae" />

You'll see the rest now floats above the plate

<img width="3188" height="2110" alt="Image" src="https://github.com/user-attachments/assets/8bfedb62-14e2-4b7d-96ad-e4f8e516521e" />

Now print that part with the settings you like (e.g: Spiral Vase mode)
